gemini/__init__.py

```diff
@@ -1,8 +0,0 @@
-import os
```

Comment:
yes, I added the itertools version, and did this instead of checking the python version supported, but I think since we bundle anaconda with 2.7+, it's safe to remove the version above.